README.md

```diff
@@ -161,7 +162,8 @@ Quick Example
 StatusBar.styleDefault
 =================
 
-Use the default statusbar (dark text, for light backgrounds).
+For Android - Automatically chooses light or dark content based on the device theme.
+For iOS - Automatically chooses light or dark content based on the user interface style.
```

Comment:
I think this can be reworded in a single sentence, the theme and the interface style are almost the same thing